File: fstcomp/examples/Modification/GPL/EdgeColoringTestOut/GPL/DescendingDegreeComparator.java

package GPL; 
import java.util.Comparator; 
public  class  DescendingDegreeComparator  implements Comparator<Vertex> {
	 public int compare(Vertex arg0, Vertex arg1) {	return arg1.getDegree() - arg0.getDegree(); }


}

File: fstcomp/examples/Modification/GPLCSharp/EdgeColoring/_mod/DescendingDegreeComparator.cs
/**
 * 
 */

/**
 * @author boxleitner
 * 
 */
public class DescendingDegreeComparator : System.Collections.Generic.IComparer<Vertex> {

    public int Compare(Vertex arg0, Vertex arg1) {
	return arg1.getDegree() - arg0.getDegree();
    }

}
